app/controllers/project.controller.js

```diff
@@ -37,37 +35,42 @@ const post = async (req, res) => {
 
 const put = async (req, res) => {
   try {
-    if (!isAuthorizedUser(req)) {
+    const project = await getProject(req, res);
+    if (project.loginId !== req.user.id) {
       res.status(403).send("You can only make changes to your own projects.");
+      return;
     }
 
     await projectService.put(req.body);
-    res.sendStatus(200);
+    res.sendStatus(204);
   } catch (err) {
     res.status(500).send(err);
   }
 };
 
 const del = async (req, res) => {
   try {
-    if (!isAuthorizedUser(req)) {
+    const project = await getProject(req, res);
+    if (project.loginId !== req.user.id) {
       res.status(403).send("You can only delete your own projects.");
+      return;
     }
 
     await projectService.del(req.user.id, req.params.id);
-    res.sendStatus(200);
+    res.sendStatus(204);
   } catch (err) {
     res.status(500).send(err);
   }
 };
 
 // HELPER FUNCTION:
-
-const isAuthorizedUser = req => {
-  const loginIdOfCurrentUser = req.user.id;
-  const loginIdInRequestBody = req.body.loginId;
-
-  return loginIdOfCurrentUser === loginIdInRequestBody ? true : false;
+const getProject = async (req, res) => {
+  const project = await projectService.getById(req.user.id, req.params.id); //TODO: Check on purpose of sending req.user.id
```

Comment:
fyi. req.user.id is used by the sproc to filter for projects created by that user.